src/algebra/order/monoid.lean

```diff
@@ -1039,3 +1039,23 @@ def order_embedding.mul_right
   {α : Type*} [has_mul α] [linear_order α] [covariant_class α α (swap (*)) (<)] (m : α) :
   α ↪o α :=
 order_embedding.of_strict_mono (λ n, n * m) (λ a b w, mul_lt_mul_right' w m)
+
+namespace additive
+
+variables [partial_order α] {a b : α}
```

Comment:
This only needs a preorder. Can you move this all up to within the `typetags` section above?

Reply:
Done!